api/src/pcapi/routes/backoffice/accounts.py

```diff
@@ -51,6 +55,45 @@ def get_public_account(user_id: int) -> s11n.PublicAccount:
     return s11n.PublicAccount.from_orm(user)
 
 
+@blueprint.backoffice_blueprint.route("public_accounts/user/<int:user_id>", methods=["POST"])
+@perm_utils.permission_required(perm_models.Permissions.MANAGE_PUBLIC_ACCOUNT)
+@spectree_serialize(
+    response_model=s11n.PublicAccount,
+    on_success_status=200,
+    api=blueprint.api,
+)
+def update_public_account(user_id: int, body: s11n.PublicAccountUpdateRequest) -> s11n.PublicAccount:
+    user = users_repository.get_user_by_id(user_id)
+    if not user:
+        raise ApiErrors(errors={"user_id": "L'utilisateur n'existe pas"})
+
+    if body.phoneNumber:
+        # Waiting for clarification about phone number changes allowed or not
+        raise ApiErrors(errors={"phoneNumber": "La modification du numéro de téléphone n'est pas autorisée"})
+
+    users_api.update_user_information(
+        user,
+        first_name=body.firstName,
+        last_name=body.lastName,
+        birth_date=body.dateOfBirth,
+        id_piece_number=body.idPieceNumber,
+        address=body.address,
```

Comment:
comment on fait si l'admin veut passer à "None" une valeur ?

Reply:
Pour vider un champ, l'admin peut passer une chaîne vide, qui dans la fonction sera `not None`.

Reply:
ok. ce serait plus propre d'avoir un mécanisme où on met `None` plutôt que "", non ?